wix/atlascli.json

```diff
@@ -0,0 +1,27 @@
+{
+  "product": "Atlas CLI",
+  "company": "MongoDB, Inc.",
+  "license": "LICENSE",
+  "upgrade-code": "a995cda5-9731-4158-910c-c8d15371dbda",
+  "files": {
+    "guid": "c7b0df57-c3c1-4591-becd-f0f73a912eb8",
+    "items": [
+      "bin/atlas.exe"
+    ]
+  },
+  "env": {
+    "guid": "94faac3d-4478-431c-8497-fba55dcfb249",
+    "vars": [
+      {
+        "name": "PATH",
+        "value": "[INSTALLDIR]",
+        "permanent": "yes",
+        "system": "no",
+        "action": "set",
+        "part": "last"
+      }
+    ]
+  },
+  "shortcuts": {},
+  "choco": {}
+}
```

Comment:
@gssbzn How did you generate this file for mongocli? I am getting this error message when trying to install mongocli on a windows machine with atlascli already there. I think it may be related to the fact that this file is basically the same as the one used by mongocli. What do you think?
<img width="537" alt="Screenshot 2022-02-22 at 09 41 30" src="https://user-images.githubusercontent.com/5663078/155105387-4630faf0-297d-4a53-93d2-7d41ed201ccc.png">

Reply:
did you copy/paste the `guid`? you may need to change it, I don't remember exactly but you should check `go-msi` [docs](https://github.com/mh-cbon/go-msi#configuration-file)

Reply:
> did you copy/paste the guid? 

Yes, I did

Thanks! : )